updating/preparing_for_updates/updating-cluster-prepare-past-4.18.adoc

```diff
@@ -0,0 +1,123 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="updating-cluster-prepare-past-4.18"]
+= Preparing to update from {product-title} 4.18 to a newer version
+include::_attributes/common-attributes.adoc[]
+:context: updating-cluster-prepare-past-4.18
+
+toc::[]
+
+Before you update from {product-title} 4.18 to a newer version, learn about some of the specific concerns around {op-system-base} compute machines.
```

Comment:
```suggestion
Before you update from {product-title} 4.18 to a newer version, learn about some of the specific concerns around {op-system-base-full} compute machines.
```
Since this is the first instance of the product name on the page, we need to spell it out.